jira/client.py

```diff
@@ -2630,7 +2642,7 @@ def user(self, id, expand=None):
         :rtype: User
         """
         user = User(self._options, self._session)
-        params = {}
+        params = {'accountId': id}
```

Comment:
The "accountId" change happened a couple of years ago, but I am not sure (not that I looked for  it) what is the backward compatibility policy of this package.

So, it should be interesting not to force that upon all users.